[Microsoft Graph Security] msg-purge-ediscovery-data - update docs (#33054)
demisto/content: Packs/MicrosoftGraphSecurity/ReleaseNotes/2_2_9.md

#### Integrations

##### Microsoft Graph Security

> Updated the documentation for the ***msg-purge-ediscovery-data*** command to include only the purgeType and purgeAreas options supported by Microsoft.

Reviewer: ```suggestion
Updated the**purge_areas** and **purge_type** arguments of the ***msg-purge-ediscovery-data*** command to include only the purgeType and purgeAreas options supported by Microsoft.
```
